package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "nodemon server",
     "build": "webpack --config webpack.config.js --watch",
-    "test": "jest --silent"
+    "test": "jest --silent --coverage"
```

Comment:
How does this change things?

Reply:
It adds a table that lets us see all of our code coverage.